src/external.js

```diff
@@ -76,6 +77,17 @@ External.prototype.canBuy = function (accountInfo, options) {
   );
 };
 
+External.prototype.sellCheck = function (email, options) {
```

Comment:
If I understand this function right, it only shows sell for a percentage of Blockchain and Coinify employees. Doesn't this prevent us from rolling out this feature to a wider user base?